packages/normalize/css/index.css

```diff
@@ -4,18 +4,20 @@
 
 @font-face {
   font-family: 'PS TT Commons Roman';
-  src: url('https://s2.pluralsight.com/fonts/ps-tt-commons/v1/ps-tt-commons-variable-roman.woff')
+  src: url('https://s2.pluralsight.com/fonts/ps-tt-commons/v1/ps-tt-commons-variable-roman.woff2')
     format('woff-variations');
   font-display: swap;
+  font-weight: 100 900;
```

Comment:
Teach me about this key addition.  Why was it key to making this work?

Reply:
@jaketrent This section from [an article on css-tricks.com](https://css-tricks.com/using-css-custom-properties-to-adjust-variable-font-weights-in-dark-mode/):

> Keep in mind that, if you’re using the `@font-face` rule to load fonts, you should set a `font-weight` range for each of them at the same time:

```
@font-face {
  src: url('Highgate.woff2') format('woff2-variations');
  font-family: 'Highgate';
  font-weight: 100 900;
}
```

> If you neglect this step, some variable fonts may not properly reflect specific font-weight values in current Chromium browsers.

This fixed the problem with using `font-weight` in Chrome, but it also worked around the rgba opacity bug in Safari, which I suspect is related to the use of `font-variation-settings: "wght"`.